src/User.cpp

```diff
@@ -543,18 +543,19 @@ CString& CUser::ExpandString(const CString& sStr, CString& sRet) const {
 	CString sTime = CUtils::CTime(time(nullptr), m_sTimezone);
 
 	sRet = sStr;
-	sRet.Replace("%user%", GetUserName());
-	sRet.Replace("%defnick%", GetNick());
-	sRet.Replace("%nick%", GetNick());
 	sRet.Replace("%altnick%", GetAltNick());
+	sRet.Replace("%bindhost%", GetBindHost());
+	sRet.Replace("%defnick%", GetNick());
+	sRet.Replace("%empty%", "");
```

Comment:
I guess it's better to have than not to have, I will update it in a followup commit.